templates/components/api-server/component.js

```diff
@@ -44,6 +44,12 @@ template.server = {
         'loopback/server/models',
         '../common/models',
         './models',
+      ],
+      mixins: [
```

Comment:
> It should be consistent with what we have for models IMO

:+1: for consistency

I am not sure how to apply that rule in this particular case. What property name would you find more consistent?

This is the current prologue of the scaffolded `model-config.json` file:

```json
{
  "_meta": {
    "sources": ["loopback/common/models", "etc."],
    "mixins": ["loopback/common/mixins", "etc."]
}
```